app/models/answer.rb

```diff
@@ -15,4 +15,9 @@ def respond(response)
       Their answer was #{self.text}")
   end
 
+  def destroy_answer answer
```

Comment:
Again, do we need something explicit for the answer? Doesn't look like this is doing much...